src/main/java/com/zontik/groshiky/config/security/ModelMapperConfig.java

```diff
@@ -0,0 +1,54 @@
+package com.zontik.groshiky.config.security;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.zontik.groshiky.model.Account;
+import com.zontik.groshiky.model.Transaction;
+import com.zontik.groshiky.model.TransactionDto;
+import com.zontik.groshiky.repository.AccountRepository;
+import org.modelmapper.Converter;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import javax.annotation.PostConstruct;
+
+import java.util.TimeZone;
+
+import static org.modelmapper.config.Configuration.AccessLevel.PRIVATE;
+
+@Configuration
+public class ModelMapperConfig {
+
+    final public AccountRepository accountRepository;
+
+    @Autowired
+    public ModelMapperConfig(AccountRepository accountRepository) {
+        this.accountRepository = accountRepository;
+    }
+
+    @PostConstruct
+    @Bean
+    public ModelMapper modelMapper() {
+        ModelMapper mapper = new ModelMapper();
+        Converter<Integer, Account> transactionIdToTransaction = context -> accountRepository.findAccountById(context.getSource());
```

Comment:
What about to move different types configurations into different files (classes)?